subsys/bluetooth/host_extensions.c

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/*
+ * Purpose of this file is to add nordic-LL only vendor-specific commands.
+ * Adding them in nrf is better maintainable.
+ */
+
+#if defined(CONFIG_BT_TRANSMIT_POWER_CONTROL)
+
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/bluetooth/hci.h>
+#include <../subsys/bluetooth/host/conn_internal.h>
```

Comment:
Can we avoid including internal headers? It looks like we only need this header to get the handle of a connection, but `bt_hci_get_conn_handle` could be used for that instead.

Reply:
There were some conversations in Teams about this. Unfortunately we need the handle and the forward declaration of 'struct bt_conn' was introduced specifically to avoid exposing the handle to applications. Result was that this is the best compromise.